Library/Homebrew/formula_installer.rb

```diff
@@ -1202,6 +1202,7 @@ def pour
     tab.unused_options = []
     tab.built_as_bottle = true
     tab.poured_from_bottle = true
+    tab.loaded_from_api = formula.class.to_s.deconstantize.demodulize.start_with? "FormulaNamespaceAPI"
```

Comment:
This feels pretty implementation-detaily: could this be moved to `Formulary` or similar instead?

Reply:
How about putting this in `Formula`?

```ruby
class Formula
  delegate loaded_from_api?: :"self.class"

  class << self
    def loaded_from_api?
      to_s.deconstantize.demodulize.start_with? "FormulaNamespaceAPI"
    end
  end
end
```

Then just call `formula.loaded_from_api?`

Reply:
If you're doing it on formula level then you could probably to have an override when we create the formula subclass.

Reply:
Are you suggesting some kind of private DSL that would only be used for API loads?

Reply:
@Rylan12 IMO this should be something set inside `Formulary.load_formula_from_api` e.g. `f.loaded_from_api = true` with a `attr_accessor :loaded_from_api` or something